### PR TITLE
Setting random seed for ATen

### PR DESCRIPTION
**Pull request by eugene-kharitonov**
+ a UT

### CODE REVIEW

File: test.cpp [hunk 1]
```diff
@@ -845,7 +845,19 @@ tests["autograd/~integration/cartpole"] = []() {
 
 };
 
- return tests;
+tests["autograd/random/seed"] = []() {
```
Comment (eugene-kharitonov) [inline test.cpp]: i wonder if that is sufficient?

Comment (ebetica) [reply test.cpp]: A GPU test would be nice... Although I have a feeling it might not work reliably.

File: autograd.h [hunk 2]
```diff
@@ -475,7 +475,9 @@ AUTOGRAD_OPTIMIZER_CLASS(Adam) {
   std::unordered_map<std::string, at::Tensor> exp_avg_sq_buffer_;
 };
 
-}  // namespace autograd
+void setSeed(int seed);
```
Comment (jgehring) [inline autograd.h]: Too late, by why not have this take a `uint64_t`?

Comment (eugene-kharitonov) [reply autograd.h]: #36

**Pull request merged**